libp2p/NodeTable.h

```diff
@@ -126,14 +126,17 @@ class NodeTable : UDPSocketEvents
     /// Called by implementation which provided handler to process NodeEntryAdded/NodeEntryDropped events. Events are coalesced by type whereby old events are ignored.
     void processEvents();
 
-    /// Add node to the list of all nodes and ping it to trigger the endpoint proof.
+    /// Starts async node adding tot the node table by pinging it to trigger the endpoint proof.
```

Comment:
Typo in comment (tot) and think it could be rephrased a little bit eg Starts async add of node to the node table...